pkg/controller/volume/attachdetach/reconciler/reconciler_test.go

```diff
@@ -690,6 +691,10 @@ func Test_Run_UpdateNodeStatusFailBeforeOneVolumeDetachNodeWithReadWriteOnce(t *
 }
 
 func Test_Run_OneVolumeDetachFailNodeWithReadWriteOnce(t *testing.T) {
+	// TODO: Remove skip once https://github.com/kubernetes/kubernetes/issues/116693 is fixed.
+	if goruntime.GOOS == "windows" {
+		t.Skip("Skipping test on Windows.")
+	}
```

Comment:
https://github.com/kubernetes/kubernetes/issues/107414 is fixed by https://github.com/kubernetes/kubernetes/pull/116675 in April. So this can be removed now.